### PR TITLE
Rails 6.1 compatibilty

### PR DESCRIPTION
**Pull request by abraaomiranda**
The main repository was updated with the Rails 6.1 compatibility. We should add this to our repository as well

### CODE REVIEW

File: spec/lhm/column_with_sql_spec.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 require 'spec_helper'
 
 describe Lhm::ColumnWithSql do
-  let(:name) { :some_field_name }
+  let(:name) { "some_field_name" }
```
Comment (acarolinatavaress) [inline spec/lhm/column_with_sql_spec.rb:4]: 💅 
```suggestion
  let(:name) { 'some_field_name' }
```

**Review by acarolinatavaress — APPROVED** [review]

**Comment by acarolinatavaress** [comment]
LGTM, just left a small comment

**Review by mariuslpop — APPROVED** [review]
❤️

**Review by bschettino — APPROVED** [review]
Nice job 👏

**Review by dbalexandre — APPROVED** [review]

**Pull request merged**